docs/presentation/content/RelatedPropertiesSpecification.md

```diff
@@ -172,3 +174,38 @@ The attribute allows loading additional related properties that are related to t
 ```
 
 ![Example of using the "nested related properties" attribute](./media/relatedpropertiesspecification-with-nestedrelatedproperties-attribute.png)
+
+### Attribute: `relationshipProperties`
+
+List of names or definitions of relationship class properties that should be included in the content. In addition, a couple of special values are allowed:
```

Comment:
I think it's important to mention that we're always using the last relationship in the given relationship path in `propertiesSource` (important when the path is more than 1 step long).

Reply:
Not sure I like the change. For me something like this sounds better:

> List of names or definitions of relationship class properties that should be included in the content. The properties are taken from the relationship used in the last step of the relationship path specified in the [`propertiesSource`](#attribute-propertiessource). 

@roluk, what's your opinion?

Reply:
I agree with your judgement, but I provided a shorter alternative in another comment.